frontend/packages/ui/src/Layouts/PageLayout.tsx

```diff
@@ -150,6 +163,8 @@ export const Page: PageLayout = Object.assign(
       className,
       children,
       header,
+      tabs,
```

Comment:
This should be removed, since Tabs are actually in the header.

Reply:
Done